src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+.calendar {
+  width: $calendar-width;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-width;
+
+  padding: 10px;
+
+  @for $i from 1 through 31 {
+    &__day:nth-child(#{$i}) {
+      width: $day-width;
+      height: $day-height;
+
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      box-sizing: border-box;
+
+      border: 1px solid $day-border-color;
+
+      background-color: $day-bg-color;
+
+      transition:
+        transform 0.5s,
+        background-color 0.5s;
+
+      &:hover {
+        transform: translateY(-20px);
+
+        background-color: $day-bg-color-hovered;
+
+        cursor: pointer;
+      }
+    }
+
+    &__day:nth-child(#{$i})::before {
+      content: '#{$i}';
+
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
```

Comment:
These properties are the same for all of :nth-child, so move them outside the loop